token-lending/program/src/processor.rs

```diff
@@ -1821,6 +1836,50 @@ fn get_pyth_price(pyth_price_info: &AccountInfo, clock: &Clock) -> Result<Decima
     Ok(market_price)
 }
 
+/// Validate the reserve configs, when initializing or modifying the reserve configs
+fn validate_reserve_configs(config: &ReserveConfig) -> ProgramResult {
```

Comment:
nit: let's rename this to `validate_reserve_config` to reflect the exact same type name.  Or we can add a `validate()` function to `ReserveConfig` directly, your choice!

Reply:
I like the method approach!